recipe/meta.yaml

```diff
@@ -9,6 +9,7 @@ source:
   sha256: 55d3a42313bda0aba7b0463687caf819a970e0ba206f5ed2c23724f80d2ae0f3
   patches:
     - max_files.patch
+    - error_top.diff
 
 build:
   number: 2
```

Comment:
are we going to increase the build number here to +1000?

Reply:
Good catch. Let me fix that.

Sadly OS X is still failing :-(